src/chapters/unit-testing/exercises.rst

```diff
@@ -0,0 +1,326 @@
+Exercises: Unit Testing
```

Comment:
@jimflores5, nice work overall. Having said that, I have some radical suggestions. 1) Cut out the first two sections 2) Replace them with the "try it on your own" stuff from TDD in Action. We could talk more about how to arrange this.

TDD in Action: https://github.com/LaunchCodeEducation/intro-to-professional-web-dev/blob/unit-testing/src/chapters/unit-testing/tdd-example.rst#use-tdd-to-add-these-features

Reply:
OK. We will chat again after I push up the 2nd draft.